api/api.py

```diff
@@ -120,11 +120,54 @@ def get_route_stops_with_connections(route_id):
     return route_stops
 
 
+def get_stop(stop_id):
+    result = db.engine.execute(
+        text(
+            """
+            SELECT 
+                osm_type,
+                osm_id,
+                name,
+                local_ref,
+                has_shelter,
+                has_bench,
+                has_tactile_paving,
+                has_departures_board,
+                is_wheelchair_ok,
+                st_asGeoJSON(ST_Transform(geom, 4326)) as geojson,
+                routes_at_stop
```

Comment:
the `routes_at_stop` contains many information that are not needed anymore.

This can be cleaned in another PR, but I think we need an issue to remember to do it ;)